app/javascript/packs/hideBookmarkButtons.js

```diff
@@ -0,0 +1,12 @@
+import { getUserDataAndCsrfToken } from '@utilities/getUserDataAndCsrfToken';
+
+getUserDataAndCsrfToken().then(({ currentUser }) => {
```

Comment:
@aitchiss Please check that I am using `getUserDataAndCsrfToken` correctly here 😅

Reply:
This looks great! 🙌